applications/NXmx.nxdl.xml

```diff
@@ -80,77 +107,149 @@
+
+				<field name="data" type="NX_NUMBER" recommended="true">
+					<doc>
+						For a dimension-2 detector, the rank of the data array will be 3.
+						For a dimension-3 detector, the rank of the data array will be 4.
+						This allows for the introduction of the frame number as the
+						first index.
+					</doc>
+					<dimensions rank="dataRank">
+						<dim index="1" value="np" />
+						<dim index="2" value="i" />
+						<dim index="3" value="j" />
+						<dim index="4" value="k" required="false"/>
+					</dimensions>
+				</field>
+			</group>
+
+			<group type="NXsample">
+				<field name="name" type="NX_CHAR">
+					<doc>Descriptive name of sample</doc>
+				</field>
+
+				<field name="depends_on" type="NX_CHAR">
+					<!-- better type for paths the need to resolve -->
+					<doc>
+						This is a requirement to describe for any scan experiment.
+
+						The axis on which the sample position depends may be stored
+						anywhere, but is normally stored in the NXtransformations
+						group within the NXsample group.
+
+						If there is no goniometer, e.g. with a jet, depends_on
+						should be set to "."				
+					</doc>
+				</field>
+
+				<group type="NXtransformations" minOccurs="0">
+					<doc>
+						This is the recommended location for sample goniometer
+						and other related axes.
+
+						This is a requirement to describe for any scan experiment.
+						The reason it is optional is mainly to accommodate XFEL
+						single shot exposures.
+
+						Use of the depends_on field and the NXtransformations group is
+						strongly recommended.  As noted above this should be an absolute
+						requirement to have for any scan experiment.
+
+						The reason it is optional is mainly to accommodate XFEL
+						single shot exposures.
+					</doc>
+				</group>
+
+				<field name="temperature" units="NX_TEMPERATURE" minOccurs="0" />
+
+			</group>
+
+
 			<group type="NXinstrument">
-			        <field name="name" minOccurs="1">
-                                       <doc>Name of instrument</doc>
-                                       <attribute name="short_name">
-                                               <doc>short name for instrument, perhaps the acronym</doc>
-                                       </attribute>
-                                </field>
+				<field name="name" minOccurs="1">
+					<doc>
+						Name of instrument.  Consistency with the controlled 
+						vocabulary beamline naming in http://mmcif.wwpdb.org
+						/dictionaries/mmcif_pdbx_v50.dic/Items
+						/_diffrn_source.pdbx_synchrotron_beamline.html
+						and http://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic
+						/Items/_diffrn_source.type.html} is highly recommended.
+					</doc>
+					<attribute name="short_name">
+						<doc>Short name for instrument, perhaps the acronym.</doc>
+					</attribute>
+				</field>
+
+				<field name="time_zone" type="NX_DATE_TIME" recommended="true">
+					<doc>
+						ISO 8601 time_zone offset from UTC.
```

Comment:
Why is this needed if times above are recorded in UTC?

Reply:
For use in resolving issues related to other time dependent information